src/sentry/runner/commands/import_system_symbols.py

```diff
@@ -0,0 +1,134 @@
+    from sentry.models import DSymBundle, DSymObject, DSymSDK
+
+    sdk = DSymSDK.objects.get_or_create(
+        dsym_type=sdk_info['dsym_type'],
+        sdk_name=sdk_info['sdk_name'],
+        version_major=sdk_info['version_major'],
+        version_minor=sdk_info['version_minor'],
+        version_patchlevel=sdk_info['version_patchlevel'],
+        version_build=sdk_info['version_build'],
+    )[0]
+
+    obj = DSymObject.objects.get_or_create(
+        cpu_name=data['arch'],
+        object_path='/' + data['image'].strip('/'),
+        uuid=str(uuid),
+        vmaddr=data['vmaddr'],
+        vmsize=data['vmsize'],
+    )[0]
+
+    DSymBundle.objects.get_or_create(
+        sdk=sdk,
+        object=obj
+    )[0]
+
+    step = 4000
+    symbols = data['symbols']
+    for idx in xrange(0, len(symbols) + step, step):
+        end_idx = min(idx + step, len(symbols))
+        yield [{
+            'object_id': obj.id,
+            'address': symbols[x][0],
+            'symbol': symbols[x][1],
+        } for x in xrange(idx, end_idx)]
+
+
+def process_archive(members, zip, sdk_info, threads):
+    q = Queue.Queue(threads)
+
+    def process_items():
+        cur = connection.cursor()
+        cur.execute('begin')
+        cur.execute('''
+            prepare add_sym(bigint, bigint, text) as
+                insert into sentry_dsymsymbol (object_id, address, symbol)
+                select $1, $2, $3
+                where not exists (select 1 from sentry_dsymsymbol
+                    where object_id = $1 and address = $2);
+        ''')
+        while 1:
+            items = q.get()
+            if items is SHUTDOWN:
+                break
+            try:
+                cur.executemany('''
+                    execute add_sym(%(object_id)s, %(address)s, %(symbol)s);
+                ''', items)
+            except IntegrityError:
+                print 'warning: already seen'
+                continue
+        cur.execute('commit')
+
+    pool = []
+    for x in xrange(threads):
+        t = threading.Thread(target=process_items)
+        t.setDaemon(True)
+        t.start()
+        pool.append(t)
+
+    for member in members:
+        try:
+            id = uuid.UUID(member)
+        except ValueError:
+            continue
+        for chunk in load_bundle(q.put, id, json.load(zip.open(member)),
+                                 sdk_info):
+            q.put(chunk)
+
+    for t in pool:
+        q.put(SHUTDOWN)
+    for t in pool:
+        t.join()
+
+
+@click.command(name='import-system-symbols',
+               short_help='Import system debug symbols.')
+@click.argument('bundles', type=click.Path(), nargs=-1)
+@click.option('--sdk', default='iOS', help='The SDK identifier')
+@click.option('--dsym-type', default='macho', help='The type of the symbol')
+@click.option('--threads', default=8, help='The number of threads to use')
+@configuration
+def import_system_symbols(bundles, sdk, dsym_type, threads):
```

Comment:
We should bury this under a `sentry dsym import_system_symbols` imo in case there are more things we need to do with them.